include/boost/numeric/ublas/tensor/fixed_rank_strides.hpp

```diff
@@ -0,0 +1,203 @@
+//
+//  Copyright (c) 2018-2020, Cem Bassoy, cem.bassoy@gmail.com
+//  Copyright (c) 2019-2020, Amit Singh, amitsingh19975@gmail.com
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Google and Fraunhofer IOSB, Ettlingen, Germany
+//
+/// \file strides.hpp Definition for the basic_strides template class
+
+
+#ifndef _BOOST_UBLAS_TENSOR_FIXED_RANK_STRIDES_HPP_
+#define _BOOST_UBLAS_TENSOR_FIXED_RANK_STRIDES_HPP_
+
+#include <vector>
+#include <limits>
+#include <numeric>
+#include <stdexcept>
+#include <initializer_list>
+#include <algorithm>
+#include <cassert>
```

Comment:
Do we need that many headers here?

Reply:
I removed them as they are included through `fixed_rank_exents.hpp`